install_config/persistent_storage/persistent_storage_flex_volume.adoc

```diff
@@ -267,13 +267,14 @@ For example, to install the FlexVolume driver for the storage `foo`, place the
 executable file at:
 *_/usr/libexec/kubernetes/kubelet-plugins/volume/exec/openshift.com~foo/foo_*.
 
-In {product-title} {product-version} since controller-manager runs as a static pod, the
-Flexvolume binary that performs attach/detach MUST be a self-contained executable with no external
-dependencies.
-
-On Atomic hosts the default location of Flexvolume plugin directory is `/etc/origin/kubelet-plugins`.
-Thus the Flexvolume executable must be placed under *_/etc/origin/kubelet-plugins/volume/exec/<vendor>~<driver>/<driver>_* on
-all master and nodes in the cluster.
+In {product-title} {product-version}, since controller-manager runs as a static
+pod, the FlexVolume binary that performs attach/detach must be a self-contained
+executable with no external dependencies.
```

Comment:
s/controller-manager/\*controller-manager*
s/binary/binary file
s/"attach/detach"/the attach and detach operations (or similar - remove the / and add a noun)
s/executable/executable file